pkg/operator/ceph/csi/spec.go

```diff
@@ -91,6 +91,8 @@ type Param struct {
 	CephFSLivenessMetricsPort                uint16
 	CSIAddonsPort                            uint16
 	RBDLivenessMetricsPort                   uint16
+	KubeApiBurst                             uint16
+	KubeApiQPS                               uint16
```

Comment:
use float32 for QPS?

Reply:
ok, should consistent with the parameter types provided by CSI.